src/main/java/org/reaktivity/nukleus/mqtt/internal/MqttConfiguration.java

```diff
@@ -47,6 +50,11 @@ public long getPublishTimeout()
         return PUBLISH_TIMEOUT.get(this);
     }
 
+    public long getConnectTimeout()
```

Comment:
Please remove the `get` prefix from all configuration methods.
So, `connectTimeout()`, `clientId()`, etc.